acceptance/tests/parser_functions/lookup.rb

```diff
@@ -90,6 +89,36 @@ def data()
   end
 end
 
+def mod_manifest_metadata_json(module_name = nil, testdir)
+  if module_name
+    metadata_manifest = <<PPmetadata
+      file { '#{testdir}/environments/production/modules/#{module_name}/metadata.json':
+        ensure => file,
+        content => '
+{
+  "name": "tester-#{module_name}",
+  "version": "0.1.0",
+  "author": "tester",
+  "summary": null,
+  "license": "Apache-2.0",
+  "source": "",
+  "project_page": null,
+  "issues_url": null,
+  "dependencies": [
+  ],
+  "data_provider": "function"
+}
+        ',
+        mode => "0644",
+      }
+      file { '#{testdir}/environments/production/modules/#{module_name}/lib/puppet/bindings':
+        ensure => absent,
+	force  => true,
```

Comment:
small whitespace issue.  heredocs have lots of whitespace indentation issues, but we can still be consistent.